docs/blockchain-rpc/hedera-rpc.md

```diff
@@ -0,0 +1,177 @@
+      "type": "CryptoTransfer",
+      "bytes": "Co8BKowBCocBChoKDAjchKWmBhDW..."
+    }
+  }
+}
+```
+
+#### Result
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "response": {
+      "nodeId": "0.0.3",
+      "transactionHash": "252b8fd...",
+      "transactionId": "0.0.12345678@1689281510.675369303"
+    },
+    "receipt": {
+      "status": {
+        "_code": 22
+      },
+      "accountId": null
+      // ...Truncated -- see note about TransactionReceipt above
+    }
+  }
+}
+```
+
+## hedera_signAndReturnTransaction
+
+The `hedera_signAndReturnTransaction` method is similar to the `hedera_signAndExecuteTransaction` except that instead of the wallet signing and submitting the
+transaction to the Hedera network, the wallet just signs the transaction and returns the signed transaction to the dApp for further handling.
+
+### Parameters
+
+    1. `Object` - signAndExecuteTransaction parameters
+      1.1 `transaction` : `Object` - transaction parameters
+        1.1.1 `type` : `String` - The Hedera function name (*1)
+        1.1.2 `bytes` : `String` - Transaction that has been converted to bytes array (*2) and encoded as a base64 string
+
+\*1. The full list of functions can be found [here](https://hashgraph.github.io/hedera-protobufs/#proto.HederaFunctionality), and are available via the [`RequestType` class](https://github.com/hashgraph/hedera-sdk-js/blob/50d7f76ffd1fd797a029192fab08b035720998b9/src/RequestType.js) from the Hedera SDK.
+
+\*2. Transactions generated with the Hedera SDK can be converted to bytes with the [`toBytes` method](https://github.com/hashgraph/hedera-sdk-js/blob/50d7f76ffd1fd797a029192fab08b035720998b9/src/transaction/Transaction.js#L1015).
+
+### Returns
+
+    1. `Object` - signAndExecuteTransaction parameters
+      1.1 `transaction` : `Object` - transaction parameters
+        1.1.1 `type` : `String` - The Hedera function name (*1)
+        1.1.2 `bytes` : `String` - Transaction that has been converted to bytes array (*2) and encoded as a base64 string
+
+### Example
+
+#### Request
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "hedera_signAndExecuteTransaction",
+  "params": {
+    "transaction": {
+      "type": "CryptoTransfer",
+      "bytes": "Co8BKowBCocBChoKDAjchKWmBhDW..."
+    }
+  }
+}
+```
+
+#### Result
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "transaction": {
+      "type": "CryptoTransfer",
+      "bytes": "VGhpcyBpcyBqdXN0IHNvbWUgc3R1..."
+    }
+  }
+}
+```
+
+## hedera_signMessage
+
+The `hedera_signMessage` method is used to pass arbitrary data to the wallet to be signed with the user's private key. The resulting signed message is then passed back to the dApp.
+
+### Parameters
+
+    1. `Object` - signMessage parameters
+      1.1 `message` : `string` - base64 encoded string of arbitrary data
+
+### Returns
+
+    1. `Object` - signMessage result
+      1.1 `signature` : `string` - base64 encoded string of signed message bytes array
+
```

Comment:
Perhaps a SigMap would be a better choice here because it also contains necessary information required by the network for fusing mutliple signatures required for a transaction.  Additionally the controller can return multiple signatures if required.

Reply:
okay, I'll dive into this. thanks!